_articles/use-docking-station.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: article
+title: Display - Docking Station Support
+description: >
+  A Docking Station makes it easy to use multi monitors. It also adds an ethernet port, multi USB ports and more while only using one USB 3.0 port on your laptop.
+keywords:
+  - docking 
+  - station
+  - popos
+  - ubuntu
+  - System76
+image: https://system76.com/images/system76.png
+hidden: false
+section: articles
+---
+
+### Compatible Intel systems
+
+ - Lemur
+ - Galago Pro
+ - Gazelle
+
+### Compatible NVIDIA systems
+
+ - Oryx Pro
+ - Serval WS
+ - Bonoobo WS
+
+### Product and Model:
+
+We have tested:
+ - [Plugable UD-3900](http://plugable.com/products/ud-3900/) [works with NVIDIA and Intel systems]
+
+### For Intel systems
+
+You'll also need to install the 'dkms' package to install DisplayLink Driver. The NVIDIA Driver install this package and has it already.
+
+```
+sudo apt install dkms
+```
+
+### Pop!_OS
```

Comment:
I'd recommend combining these 2 sections, and title it:

### DisplayLink Driver

And talk about installing the driver.  You can mention that NVIDIA cards need the closed source driver to function, and how to install it.